### PR TITLE
Budget App Added

### LINKED ISSUE #274: Budgeting App
## 🚀 Feature

Budgeting app - A all in one solution to all your financial problems

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes I have gone through the CONTRIBUTING.md file .

## Motivation

We often ignore our expenditure in the starting of the month and end up spending more than we were supposed to . So this app will be a solution to this problem and will help us in planning our monthly expense.

## Pitch

This app will help people in becoming their own financial advisor and stop them from spending on unnecessary goods by giving them a complete foresight of how much money they are spending on different stuffs and what will be their leftover balance in the end of the month with their current expenditure rate.

After this , it will calculate the amount of money they can spend on different things (such as food , savings , entertainment , travel etc) in a tabular manner.

In the end this will also give them a PRO-TIP based on their expense so that manage their finance more efficiently in the future.

Issue made under GSSoC'22 contributions.

### PR DESCRIPTION
**Pull request by Kunal0220**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes I have read the Contributing Guidelines.

### Description

1. This is a simple budget app which takes your monthly salary as input and your expenditure (in %) for different things.
2. Once we click on calculate , It will show a table with the amount of money you can spend on each item.
3. It will give an error if people spend more than they earn.

Fixes #274 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
issue #274 

[Contribution made under GSSoC 22']

Video Walkthrough:

https://user-images.githubusercontent.com/71934168/162234914-b33d1175-ce97-41e6-83b1-fa394965b049.mp4

### CODE REVIEW

**Review by photon149 — APPROVED** [review]
@Kunal0220 Well executed.
No duplicity ✅ 
Approved from my end.
@Kushal997-das take a look

File: Web Development/Basic/Budgeting App/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Budgeting-Website
+This is a basic budgeting app which takes the salary as input and expenditure in percent , and gives back the amount of money you can spend in numbers.
+
+<!-- Screenshot -->
+<img width="1385" alt="Screenshot 2022-04-02 at 12 04 44 AM" src="https://user-images.githubusercontent.com/71934168/161324569-912f458c-ba5e-4f88-b3cc-1f211d0d0347.png">
```
Comment (Kushal997-das) [inline Web Development/Basic/Budgeting App/README.md:5]: Reduce the width size to less than `700`

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Kunal0220

**Comment by Kunal0220** [comment]
I made the change , Can you please review it once ?

**Review by Kushal997-das — APPROVED** [review]
@Kunal0220 Great work !

**Pull request merged**